kong/cmd/utils/prefix_handler.lua

```diff
@@ -120,6 +121,36 @@ local function compile_conf(kong_config, conf_template)
   end)
 end
 
+local function write_env_file(path, data)
+  local c = require "lua_system_constants"
+
+  local flags = bit.bor(c.O_CREAT(), c.O_WRONLY())
+  local mode  = bit.bor(c.S_IRUSR(), c.S_IWUSR(), c.S_IRGRP())
+
+  local fd = ffi.C.open(path, flags, mode)
+  if fd < 0 then
+    local errno = ffi.errno()
+    return nil, "Unable to open env path " .. path .. " (" ..
```

Comment:
Ok, this is going to be very pedantic, but: this is the only capitalized error message returned by this function. Other messages are lower-cased (as are or should be all other error messages in the codebase).

I said it would be pedantic! 😁

Reply:
Fixed! Nice :eyes:. We should indeed be consistent. Solid review :+1: